src/renderers/webxr/WebXRManager.js

```diff
@@ -301,6 +301,13 @@ function WebXRManager( renderer, gl ) {
 
 	this.getCamera = function ( camera ) {
 
+		cameraVR.near = cameraR.near = cameraL.near = camera.near;
+		cameraVR.far = cameraR.far = cameraL.far = camera.far;
+		session.updateRenderState( {
```

Comment:
Instead of always doing the API call, you could cache `depthNear` and `depthFar` and only call the method if these values are different to `cameraVR.near` and `cameraVR.far`.

This state caching is also used in the renderer. I would call the cache variables: `_currentDepthNear` and `_currentDepthFar` and then do this:

```js
if ( _currentDepthNear !== cameraVR.near || _currentDepthFar !== cameraVR.far ) {

    session.updateRenderState( {
        depthNear: cameraVR.near,
        depthFar: cameraVR.far
    } );

    _currentDepthNear = cameraVR.near;
    _currentDepthFar = cameraVR.far;

}
```